src/style.css

```
@@ -1,3 +1,78 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
 body {
   margin: 0;
+
+  --item-color: #00acdc;
```

Create a variable with root selector
```suggestion
  --main-color: #00acdc;
```